gcc/rust/typecheck/rust-hir-type-check-pattern.cc

```diff
@@ -139,6 +139,15 @@ TypeCheckPattern::visit (HIR::TupleStructPattern &pattern)
     }
 }
 
+void
+emit_invalid_field_error (Location loc, Rust::TyTy::VariantDef *variant,
+			  const char *message)
```

Comment:
```suggestion
			  const std::string &name)
```

Reply:
I was thinking about this. Thanks @CohenArthur :)